content/kubernetes/re-clusters/redis-on-flash.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: Use Redis on Flash on Kubernetes
+linkTitle: Redis on Flash
+description: Deploy and configure a Redis cluster or database on Kubernetes that supports Redis on Flash.
+weight: 16
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/redis-on-flash.md,
+    /kubernetes/re-clusters/redis-on-flash/,
+]  
+---
+
+## Prerequisites
+
+Redis Enterprise Software for Kubernetes supports using NVMe (nonvolatile memory express) SSDs (solid state drives) for flash storage with the Redis on Flash feature.
+
+Before creating your Redis clusters or databases, these SSDs must be:
+
+- [locally attached to worker nodes in your Kubernetes cluster](https://kubernetes.io/docs/concepts/storage/volumes/#local)
+- [formatted and mounted]() on the worker nodes that will run Redis Enterprise pods
```

Comment:
```suggestion
- [formatted and mounted]() on the nodes that will run Redis Enterprise pods
```

Reply:
I think it's important to specify the type of nodes here, but I'll check with R&D to make sure. 
@heinrich-redislabs @laurentdroin @bascar do you know if we need to specify these are mounted on worker nodes? Or can we just say nodes?

Reply:
I think in that case it's enough to say "nodes". Master nodes will not run Redis Enterprise pods.
In fact, not all worker nodes need to have locally attached SSDs, only a subset that will run RE pods.
So just saying "nodes" is correct.